services/dask-sidecar/requirements/constraints.txt

```diff
@@ -17,3 +17,17 @@
 
 # allows to put files in the HTTP file system
 fsspec>=2021.10.1
+
+
+
+#Could not find a version that matches tornado<6.2,>=5.1,>=6.0.3,>=6.1.0,>=6.2 (from dask-gateway==2022.10.0->-r requirements/_base.in (line 25))
+#Tried: 0.2, 1.0, 1.1, 1.1.1, 1.2, 1.2.1, 2.0, 2.1, 2.1.1, 2.2, 2.2.1, 2.3, 2.4, 2.4.1, 3.0, 3.0.1, 3.0.2, 3.1, 3.1.1, 3.2, 3.2.1, 3.2.2, 4.0, 4.0.1, 4.0.2, 4.1, 4.2, 4.2.1, 4.3, 4.4, 4.4.1, 4.4.2, 4.4.3, 4.5, 4.5.1, 4.5.2, 4.5.3, 5.0, 5.0.1, 5.0.2, 5.1, 5.1.1, 6.0, 6.0.1, 6.0.2, 6.0.3, 6.0.4, 6.1, 6.1, 6.1, 6.2, 6.2
+#Skipped pre-versions: 4.1b2, 4.2b1, 4.3b1, 4.3b2, 4.4b1, 4.5b1, 4.5b2, 5.0a1, 5.0b1, 5.1b1, 6.0a1, 6.0b1, 6.1b1, 6.1b2, 6.1b2, 6.1b2, 6.2b1, 6.2b1, 6.2b2, 6.2b2
+#There are incompatible versions in the resolved dependencies:
+#  tornado (from dask-gateway==2022.10.0->-r requirements/_base.in (line 25))
+#  tornado<6.2,>=6.0.3 (from distributed==2022.9.2->dask-gateway==2022.10.0->-r requirements/_base.in (line 25))
+#  tornado>=5.1 (from bokeh==2.4.3->dask[complete,distributed]==2022.9.2->-r requirements/_base.in (line 26))
+#  ===> tornado>=6.2 (from jupyter-client==7.4.2->jupyter-server==1.21.0->jupyter-server-proxy==3.2.2->-r requirements/_base.in (line 27))
+#  tornado>=6.1.0 (from terminado==0.16.0->jupyter-server==1.21.0->jupyter-server-proxy==3.2.2->-r requirements/_base.in (line 27))
+#  tornado>=6.1.0 (from jupyter-server==1.21.0->jupyter-server-proxy==3.2.2->-r requirements/_base.in (line 27))
+jupyter-client<7.0.0
```

Comment:
ouch pretty annoying...

Reply:
now is gone. i targetted update dask libraries